scripts/xpcs_online_boost_client.py

```diff
@@ -136,7 +154,12 @@ def arg_parse():
         }
     }
 
-    corr_flow = XPCSBoost()
+    callback_login_manager = None
+    if CLIENT_ID and CLIENT_SECRET:
+        callback_login_manager = CallbackLoginManager({}, callback=callback)
+        
+    corr_flow = XPCSBoost(login_manager=callback_login_manager)
+
```

Comment:
Interesting, not tying this to the deployment means we could conceivably run this using other peoples deployment configurations but with client credentials. Not sure yet if that would be beneficial, but it's an option!

In the future, we might want to add a check if someone attempts to run this deployment and forgets the client/secret (I think in that case the script will attempt to login a globus user and start that way, which could be confusing for someone new). But I'm happy with the changes as-is!

Reply:
Good question. I reused the deployment for local testing as I knew my user had permission to access the endpoints. I'm not sure about adding a relationship between clients and deployments. I wonder how best to do that.